### PR TITLE
feat: rabbitmq add member leave to support scale-in

### PR DESCRIPTION
**Pull request by xuriwuyun**

### CODE REVIEW

File: addons/rabbitmq/templates/componentdefinition.yaml [hunk 1]
```diff
@@ -38,6 +38,33 @@ spec:
...
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                apiVersion: v1
+                fieldPath: metadata.namespace
+          - name: RABBITMQ_USE_LONGNAME
+            value: "true"
+          - name: K8S_SERVICE_NAME
+            value: $(KB_CLUSTER_COMP_NAME)-headless
```
Comment (Y-Rookie) [inline addons/rabbitmq/templates/componentdefinition.yaml:65]: do not use `KB_CLUSTER_COMP_NAME` builti-in env

File: addons/rabbitmq/scripts/member_leave.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```
Comment (Y-Rookie) [inline addons/rabbitmq/scripts/member_leave.sh:1]: better to add ut for script

File: addons/rabbitmq/script-ut-spec/member_leave_spec.sh [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  #    setup
+
+  #    When run source ../scripts/member_leave.sh
+  #    The output should include "member_leave.sh is already leave success"
+  #    The status should be success
+  #    The file "/tmp/rabbitmq-1_leave.success" should be exist
+  #    rm -f /tmp/rabbitmq-1_leave.success
+  #  End
+
+  #  It "当离开成功文件存在且当前节点不是离开节点时，删除成功文件并退出"
```
Comment (zjx20) [inline addons/rabbitmq/script-ut-spec/member_leave_spec.sh:124]: clean up the code.

**Review by zjx20 — APPROVED** [review]

**Review by Y-Rookie — APPROVED** [review]

**Pull request merged**